nextcord/entitlement.py

```diff
@@ -0,0 +1,87 @@
+# SPDX-License-Identifier: MIT
+
+from __future__ import annotations
+
+from datetime import datetime
+from typing import Optional
+
+from .enums import EntitlementType, SKUType
+from .flags import SKUFlags
+from .object import Object
+from .types.entitlement import SKU as SKUPayload, Entitlement as EntitlementPayload
```

Comment:
This can be in an `if TYPE_CHECKING` block like other `.types` imports elsewhere.

Reply:
done as expected